automatic/chromium/tools/chocolateyInstall.ps1

```diff
@@ -12,16 +12,16 @@
   }
   
     $packageArgs = @{
-  packageName   = '{{PackageName}}'
+  packageName   = 'Chromium'
   fileType      = 'exe'
-  url           = '{{DownloadUrl}}'
-  url64bit      = '{{DownloadUrlx64}}'
+  url           = 'https://storage.googleapis.com/chromium-browser-snapshots/Win/425094/mini_installer.exe'
+  url64bit      = 'https://storage.googleapis.com/chromium-browser-snapshots/Win_x64/425083/mini_installer.exe'
   silentArgs    = $silentArgs
   validExitCodes= @(0)
   softwareName  = 'Chromium'
-  checksum      = '{{Checksum}}'
+  checksum      = '97da1f6f835b4b6ca7156c723e897083'
   checksumType  = 'md5'

```

Comment:
Didn't know that AU had that ability. I do know that "sha256" is the more secure form, and it will be worked on in the future. I will add the check for the current provided type.